enterprise/frontend/src/embedding-sdk/components/public/index.ts

```diff
@@ -1,6 +1,3 @@
-import "ee-overrides"; // eslint-disable-line import/no-duplicates
```

Comment:
We don't need EE overrides right now because we already did something similar in the SDK, but it uses `application-colors` _after_ the SDK loads the settings instead, so it doesn't rely on hydrated JSON settings. For reference, this is what ee-overrides do: 

```ts
import "./whitelabel/overrides";
```

and

```ts
import { hasPremiumFeature } from "metabase-enterprise/settings";
import { updateColors } from "metabase-enterprise/whitelabel/lib/whitelabel";

if (hasPremiumFeature("whitelabel")) {
  updateColors();
}
```